environment.yml

```diff
@@ -37,8 +38,7 @@ dependencies:
   - wheel
   - xz
   - zlib
-  - pip
   - pip:
     - eventio~=1.0
-
-
+    - zstandard
```

Comment:
Move this out of pip install, it has a conda package and it's even the base anaconda distribution (not even conda-forge)

Reply:
Ah, nice. Sure